src/FX.GrpcClient/Instance.cs

```diff
@@ -0,0 +1,58 @@
+﻿
+using System;
+using System.Threading.Tasks;
+using Grpc.Core;
+using Grpc.Net.Client;
+using GrpcService;
+
+namespace GrpcClient
+{
+    public enum ServerEnvironment
+    {
+        Local,
+        Unknown,
+        Production,
+    }
+
+    public class Instance
+    {
+        private static string serverFqdn =
+            "localhost";
+
+        public static readonly int Port = 7178;
+
+        public FXGrpcService.FXGrpcServiceClient Connect()
+        {
+            var channel = GrpcChannel.ForAddress($"https://{serverFqdn}:{Port}");
+            var client = new FXGrpcService.FXGrpcServiceClient(channel);
+            return client;
+        }
+
+        public static ServerEnvironment ServerEnvironment {
+            get {
+                if (serverFqdn == "localhost")
+                {
+                    return ServerEnvironment.Local;
+                }
+                else if (serverFqdn.EndsWith(".runinto.me"))
```

Comment:
this logic is RunIntoMe-specific, please remove it from this PR

Reply:
@webwarrior-ws ^

Reply:
removed